synapse/handlers/room_summary.py

```diff
@@ -713,8 +715,12 @@ async def _build_room_entry(self, room_id: str, for_federation: bool) -> JsonDic
             ),
             "guest_can_join": stats["guest_access"] == "can_join",
             "room_type": create_event.content.get(EventContentFields.ROOM_TYPE),
+            "im.nheko.summary.version": stats["version"],
         }
 
+        if stats["encryption"]:
+            entry["im.nheko.summary.encryption"] = stats["encryption"]
```

Comment:
I'm not sure, if I need the if to leave fields out and if it is a better idea to leave the encryption field out completely for unencrypted rooms and if I need to do the same for the version or not, since old rooms have no version. Otoh, that wasn't done for the name and topic, so...